07_palindromes.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# Implementation for RubyMonk Primer Problem 7 Palindromes
+# Problem Statement:
+# Given a sentence, return true if the sentence is a palindrome.
+# You are supposed to write a method palindrome? to accomplish this task.
+# Note Ignore whitespace and cases of characters.
+# Example:
+# Given "Never odd or even" the method should return true
+class Palindrome
+  def palindrome?(sentence)
```

Comment:
If you pass `sentence` in at instantiation you don't need to toss it around between methods.  You could also memoize the normalized version and not have to do that computation again.

Reply:
Agreed but the original RubyMonk problem is...
```
def palindrome?(sentence)
  # Write your code here
end
```

I created the wrapper `Palindrome` class for better test style.